editor/src/components/canvas/canvas-strategies/strategies/reparent-as-static-strategy.tsx

```diff
@@ -247,6 +252,41 @@ function applyStaticReparent(
               ]
               duplicatedElementNewUids = placeholderResult.duplicatedElementNewUids
 
+              commonPatches.push(
+                setActiveFrames(
+                  filteredSelectedElements.map((path) => {
+                    const source = zeroRectIfNullOrInfinity(
+                      MetadataUtils.getFrameOrZeroRectInCanvasCoords(
+                        path,
+                        canvasState.startingMetadata,
+                      ),
+                    )
+                    return {
+                      action: 'reparent',
+                      target: activeFrameTargetRect(
+                        canvasRectangle({
+                          x:
+                            interactionSession.interactionData.type === 'DRAG'
```

Comment:
Could we pull this out into some variables as it looks _really_ busy?

Reply:
https://github.com/concrete-utopia/utopia/pull/4538/commits/35e62b2f2ba2ef3c587719d1b58a5cb598f626f4